vradzin/02_HTML/task3.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Меню</TITLE> 
+    </HEAD>  
+    <BODY> 
+    <h2>Горизонтальное меню</h2>
+    <div >
+        <a href="#">Главная</a>
+        <a href="#">Каталог</a>
+        <a href="#">Акции</a>
+        <a href="#">О нас</a>
+    </div>
+    <h2>Меню список</h2>
+    <ul>
+        <li><a href="#">Главная</a></li>
+        <li><a href="#">Каталог</a></li>
+        <li><a href="#">Акции</a></li>
+        <li><a href="#">О нас</a></li>
+      </ul>
+    <h2>Меню с вложенным списком</h2>
+    <ul>
```

Comment:
Wrong list structure, could you please check all these section

Reply:
made a correction